<launch>
    <node name="toGoal" pkg="deneme" type="move.py" output="screen"/>
</launch>